superchain/configs/chainList.toml

```diff
@@ -0,0 +1,175 @@
+[[chains]]
+  name = "Base"
+  identifier = "mainnet/base"
+  chainId = 8453
+  rpc = ["https://mainnet.base.org"]
+  explorers = ["https://explorer.base.org"]
+  superchain_level = 2
+  [chains.parent]
+    type = "L2"
+    chain = "mainnet"
```

Comment:
Interesting, didn't know that toml can be written with such indentation. The [official TOML website](https://toml.io/en/v1.0.0#array-of-tables) doesn't indent. But I guess since it's just produced by the toml go package, it's also correct.